example-get-started-experiments/generate.sh

```diff
@@ -109,14 +108,18 @@ dvc remove models/model.pkl.dvc
 dvc stage add -n train \
   -p base,train \
   -d src/train.py -d data/train_data \
-  -o models/model.pkl \
+  -o models/model.pkl -o models/model.pth \
   python src/train.py
 
 dvc stage add -n evaluate \
   -p base,evaluate \
   -d src/evaluate.py -d models/model.pkl -d data/test_data \
   python src/evaluate.py
 
+dvc stage add -n sagemaker \
+  -d models/model.pth -o model.tar.gz \
+  'cp models/model.pth sagemaker/code/model.pth && cd sagemaker && tar -cpzf model.tar.gz code/ && cd .. && mv sagemaker/model.tar.gz .  && rm sagemaker/code/model.pth'
```

Comment:
Some minor thoughts to try to clean this up a bit:

I haven't tested this, but wondering if we can simplify this at all with something like this?

```suggestion
  'cp models/model.pth sagemaker/out && cp sagemaker/code sagemaker/out && tar -cpzf model.tar.gz sagemaker/out'
```

Also, I wonder if it would be better to append directly to `dvc.yaml` and use list syntax for `cmd` instead of `&&`?

Reply:
> Also, I wonder if it would be better to append directly to dvc.yaml and use list syntax for cmd instead of &&?

I didn't manage to use list cmd with `stage add` (which we use in the `generate.sh` script , but might be missing something